isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp

```diff
@@ -27,10 +27,17 @@ void IsisMain() {
   try {
     ProcessImport importer;
     translateCoreInfo(xmlFileName, importer);
-    // fails 
-    importer.SetInputFile(xmlFileName.removeExtension().addExtension("dat").expanded());
+    
+    // Need to check for original file extension as well as file extension from tgocassisrdrgen
```

Comment:
This works and I've used this pattern in my own code, but as a suggestion, I think something like the following would make it more clear that `.img` and `.dat` are both expected inputs to `tgocassis2isis`. The `try/catch` implies in some sense that `.dat` is good input  and `.img` is bad input. See: https://isocpp.org/wiki/faq/exceptions#exceptions-separate-good-and-bad-path 

untested pseudocode: 
```
if ( xmlFileName.removeExtension().addExtension("dat").expanded().fileExists() ) {
   importer.SetInputFile(xmlFileName.removeExtension().addExtension("dat").expanded());
}
else if ( importer.SetInputFile(xmlFileName.removeExtension().addExtension("img").expanded().fileExists() ) {
  importer.SetInputFile(xmlFileName.removeExtension().addExtension("img").expanded());
}
else {
   // throw error about being unable to open input file
}
```

I remember you saying during the meeting on Friday that there was something complicated about this situation that perhaps makes my above suggestion untenable.... so if that's the case, never mind ! :)

Reply:
Thank you for feedback on this one! I completely agree. Will get that changed.  :-)